bundles/af-core/src/main/java/com/adobe/cq/forms/core/components/internal/models/v1/form/HCaptchaImpl.java

```diff
@@ -0,0 +1,121 @@
+ ~ See the License for the specific language governing permissions and
+ ~ limitations under the License.
+ ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~*/
+package com.adobe.cq.forms.core.components.internal.models.v1.form;
+
+import java.util.LinkedHashMap;
+import java.util.Map;
+
+import javax.inject.Inject;
+import javax.inject.Named;
+
+import org.apache.sling.api.SlingHttpServletRequest;
+import org.apache.sling.api.resource.Resource;
+import org.apache.sling.api.resource.ResourceResolver;
+import org.apache.sling.models.annotations.Exporter;
+import org.apache.sling.models.annotations.Model;
+import org.apache.sling.models.annotations.injectorspecific.InjectionStrategy;
+import org.apache.sling.models.annotations.injectorspecific.OSGiService;
+import org.apache.sling.models.annotations.injectorspecific.ValueMapValue;
+import org.osgi.service.component.annotations.Reference;
+
+import com.adobe.aemds.guide.model.HCaptchaConfiguration;
+import com.adobe.aemds.guide.service.CloudConfigurationProvider;
+import com.adobe.aemds.guide.service.GuideException;
+import com.adobe.cq.export.json.ComponentExporter;
+import com.adobe.cq.export.json.ExporterConstants;
+import com.adobe.cq.forms.core.components.internal.form.FormConstants;
+import com.adobe.cq.forms.core.components.models.form.HCaptcha;
+import com.adobe.cq.forms.core.components.util.AbstractCaptchaImpl;
+import com.fasterxml.jackson.annotation.JsonIgnore;
+
+@Model(
+    adaptables = { SlingHttpServletRequest.class, Resource.class },
+    adapters = { HCaptcha.class,
+        ComponentExporter.class },
+    resourceType = { FormConstants.RT_FD_FORM_HCAPTCHA_V1 })
+@Exporter(name = ExporterConstants.SLING_MODEL_EXPORTER_NAME, extensions = ExporterConstants.SLING_MODEL_EXTENSION)
+public class HCaptchaImpl extends AbstractCaptchaImpl implements HCaptcha {
+
+    @Inject
+    private ResourceResolver resourceResolver;
+
+    private Resource resource;
+
+    @Reference
+    @ValueMapValue(injectionStrategy = InjectionStrategy.OPTIONAL)
+    private HCaptchaConfiguration hCaptchaConfiguration;
+
+    @OSGiService
+    @ValueMapValue(injectionStrategy = InjectionStrategy.OPTIONAL)
+    private CloudConfigurationProvider cloudConfigurationProvider;
+
+    @ValueMapValue(injectionStrategy = InjectionStrategy.OPTIONAL)
+    @JsonIgnore
+    @Named("cloudServicePath")
+    protected String cloudServicePath;
+
+    @ValueMapValue(injectionStrategy = InjectionStrategy.OPTIONAL)
+    @JsonIgnore
+    @Named("size")
+    protected String size;
+
+    private static final String SITE_KEY = "siteKey";
+    private static final String URI = "uri";
+    private static final String SIZE = "size";
+    private static final String THEME = "theme";
+    private static final String TYPE = "type";
+
+    @Override
+    @JsonIgnore
+    public String getCloudServicePath() {
+        return cloudServicePath;
+    }
+
+    @JsonIgnore
+    public String getSize() {
```

Comment:
You don't need a getter since this is not overrridden and also not part of interface.